global.js

```diff
@@ -1,9 +0,0 @@
-require('node-libs-react-native/globals')
-
-global.navigator.userAgent = ''
-const navigatorCopy = { ...global.navigator }
-Object.defineProperty(global, 'navigator', {
-  set: () => {},
-  get: () => navigatorCopy
-});
-
```

Comment:
why was this removed?

Reply:
this file isn't used anymore
it doesn't included anywhere and isn't referenced from the any config (webpack, jest, metro etc)